src/esf-mail/esf-mail-notification.ts

```diff
@@ -45,10 +67,11 @@ export function setupEsfNotificationMail(
   });
 
   /**
-  * Sqs Queue: receives messages (CSV) from esb.
+  * Sqs Queue (fifo): receives messages (CSV) from esb.
   */
-  const esfMailNotificationSQSqueue = new sqs.Queue(scope, 'esf-mail-notification-sqs', {
+  const esfMailNotificationSQSqueue = new sqs.Queue(scope, 'esf-mail-notification-sqs-fifo', {
     encryption: sqs.QueueEncryption.KMS_MANAGED,
+    fifo: true, // To enable exactly-once processing
```

Comment:
Staat het nog op de planning om de mail lambda een lookup te laten doen in de S3 bucket? Dan hoeft deze geen fifo queue te zijn, net iets robuster.

Reply:
Ja, maar laat het voor nu zo staan